conf/machine/imx6qdlsabreauto.conf

```diff
@@ -42,4 +42,4 @@ SERIAL_CONSOLES = "115200;ttymxc3"
 
 MACHINE_FIRMWARE_append_mx6 = " linux-firmware-ath6k"
 
-MACHINE_FEATURES += " pci wifi bluetooth"
+MACHINE_FEATURES += "pci wifi bluetooth optee"
```

Comment:
I wonder if this works when using the community distribution due to the difference between the U-Boot in use. Did you try it?

Reply:
I did not.

Reply:
I am afraid it won't work, as community U-Boot does not support optee as far as I know.

Reply:
Okay, shall I add it with use-fsl-bsp qualification?

Reply:
Yep!